manifest_prod.yaml

```diff
@@ -11,7 +11,7 @@ applications:
     ENV: PRODUCTION
     AUTH_RELYING_PARTY_ID: "crt-portal-django-prod.app.cloud.gov"
     AUTH_AUDIENCE: "microsoft:identityserver:crt-portal-django-prod.app.cloud.gov"
-    NEW_RELIC_CONFIG_FILE: /home/vcap/app/newrelic.ini
+    NEW_RELIC_CONFIG_FILE: /root/code/newrelic.ini
```

Comment:
Do we have any documentation re: this path? (Would like to have this in the future, should the these paths ever change)

Reply:
@cantsin @Jkrzy this may be helpful https://docs.newrelic.com/docs/agents/python-agent/configuration/python-agent-configuration#agent-configuration-file.  Should we use /vacap/ since most of these are under the VCAP services @Jkrzy ?

Reply:
Reverted to `/home/vcap/app/newrelic.ini` which is the correct path in-use elsewhere when reading files pushed up to cloud.ogv